### ISSUE TITLE
P5.js not launching

### ISSUE THREAD

**Issue by Choirmaestro** [processing/p5.js#2155]
#### Nature of issue?

- On attempt of loading the file, I am greeted with an error message that states:

Script: C:\Users\...\p5\p5.js
Line: 45965
Char: 1
Error: Expected identifier, string or number.
Code: 800A404
Source: Microsoft JScript compilation error.

#### Which platform were you using when you encountered this?

- [x] Desktop/Laptop: custom-built PC that has run p5.js in the past.

#### Details about the bug: 

I opened up the p5.js version 0.5.14 file using Notepad++ just to look at it (didn't actually meddle with anything), and the line seems to be the close-curly bracket for an error code. Specifically: 
`var errorCases = {
  '0': {
    fileType: 'image',
    method: 'loadImage',
    message: ' hosting the image online,'
  },
  '1': {
    fileType: 'XML file',
    method: 'loadXML'
  },
  '2': {
    fileType: 'table file',
    method: 'loadTable'
  },
  '3': {
    fileType: 'text file',
    method: 'loadStrings'
  },
  '4': {
    fileType: 'font',
    method: 'loadFont',
    message: ' hosting the font online,'
  },
}; `
The final }; is the line that is producing the error message.

I've run p5.js quite extensively on this computer in the past, but deleted it when I thought I wouldn't be pursuing programming any longer (obviously now it's apparent that isn't true, haha.)

I'm running on a Windows 10 build 15063.540 using Chrome version 60.0.3112.113.
I also am running on freshly updated drivers for everything, as well as freshly updated Java Version 8 Update 144 (had an issue with a game not running a few days ago and had to solve it by updating things).
I attempted to run older versions of p5.js to see if that fixed the issue, but to no avail.

I'd really like to continue to use this programming environment, as I found it very easy-to-use when I was learning before, as I am now.

Thanks so much!!!

**Comment by sakshamsaxena** [comment]
There seems to be an extra comma before the final }. Try removing it locally and hitting it up again? Will look into the source meanwhile

**Comment by Choirmaestro** [comment]
I cleared out the comma and saved it, ran it, and it produced a similar error on line 46473, char 22, "expected identifier" which char 22 is a "c" in the code 
`p5.Element.prototype.class = function(c) {
  if (arguments.length === 0) {
    return this.elt.className;
  } else {
    this.elt.className = c;
    return this;
  }
};`

**Comment by sakshamsaxena** [comment]
Can you explain how are you using this script file though? And how do you aim to? I think Microsoft JScript compilation happened when you double click the file itself in explorer. Is that what you did? I'm not sure if we're prepared for such scenarios, yet

**Comment by Choirmaestro** [comment]
I opened the p5.js file in the window explorer by double-clicking on it, right-clicking and clicking "open."
is there another way to open it? in the past there was a shortcut to an .exe file that was on my desktop, however I see no option to now...
I found this program from [https://www.youtube.com/watch?v=8j0UDiN7my4&t=360s](url) 
(can't remember his name)
I downloaded the p5.js complete package from the p5js.org website, and am mainly just hoping to use it to practice learning to code JS. Did I stumble somwhere I'm not supposed to be? haha...
tl;dr: I double-clicked the unzipped file named "p5.js"
